t/00.Dmarc.t

```diff
@@ -41,6 +41,26 @@ test_setter_values();
 test_spf();
 test_dkim();
 
+{ # config file loaded before any other attr initialization
```

Comment:
There's already a couple other package declarations in this file. Is there a reason not to place this one at the bottom of the file with the rest of them?

Also, is there a reason not to put this test into an aptly named sub as all the other tests in this file are?